akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/MiscDirectivesSpec.scala

```diff
@@ -100,6 +100,23 @@ class MiscDirectivesSpec extends RoutingSpec {
       }
     }
 
+    "apply if form data is consumed" in {
```

Comment:
perhaps say "if form data is fully consumed into a map", since I believe the other form directives don't need to strictify the incoming request. Would actually be interesting to see what happens in that case.